src/bundle.rs

```diff
@@ -417,7 +420,7 @@ enum RevPot {
 struct IonChannel {
     name: String,
     reversal_potential: RevPot,
-    conductance: Quantity,
+    conductance: Option<Quantity>,
```

Comment:
Hmn. The two options here are really 'undefined' and 'given as Quantity'.. which does map perfectly to Option. Hope you're happy with a wrapper type that's basically an Option :)

Reply:
```xml
  <xs:complexType name="ChannelDensityNonUniform">
    <xs:annotation>
      <xs:documentation>Specifies a time varying ohmic conductance density, which is distributed on a region of the **cell.**  The conductance density of the channel is not uniform, but is set using the  **variableParameter** . Note, there is no dynamical description of this in LEMS yet, as this type only makes sense for multicompartmental cells. A ComponentType for this needs to be present to enable export of NeuroML 2 multicompartmental cells via LEMS/jNeuroML to NEURON
\n
:param erev: The reversal potential of the current produced
:type erev: voltage

</xs:documentation>
    </xs:annotation>
    <xs:complexContent>
      <xs:extension base="Base">
        <xs:sequence>
          <xs:element name="variableParameter" type="VariableParameter" minOccurs="0" maxOccurs="unbounded"/>
        </xs:sequence>
        <xs:attribute name="ionChannel" type="NmlId" use="required"/>
        <xs:attribute name="erev" type="Nml2Quantity_voltage" use="required"/>
        <xs:attribute name="ion" type="NmlId" use="required">
          <xs:annotation>
            </xs:annotation>
        </xs:attribute>
      </xs:extension>
    </xs:complexContent>
  </xs:complexType>
  ```
  
  TBH I don't understand why the variable parameter can point to any random parameter but you don't point it to conductance you are forced to use the default conductance value

Reply:
https://github.com/thorstenhater/nmlcc/pull/30/commits/0b34b91d88745c09e521bcb609ca909a3e1866da

Reply:
So, no need to make an option with a new name. If, however, we can express three states 
1. we have no clue
2. we have a known value
3. we have an inhomogeneous value

as an enum, I think we might save ourselves some headaches. Not all of them might be valid
NML2, though.

Reply:
The annoying thing here is that `we have an inhomogeneous value` is orthogonal to the other two (because the inhomogeneous value could be any parameter but 1 and 2 are about conductance) - except that the NML standard forces 1. in the case of 3...

Reply:
Are you happy with the current solution?